### DISCUSSION TITLE
Альтернативный режим панели файлов

### DISCUSSION THREAD

**Discussion by bigvax** [elfmz/far2l#1300] [Show and tell]
Привет всем!
Сделал тут для себя, привычный еще из mc, режим панели файлов - переделывал «Альтернативный полный».
Может кому пригодится.
```
[Panel/ViewModes/Mode0]
AlignExtensions=1
ColumnWidths=0,6,6,6,0
Columns=N,S,U,O,A
FileLowerCase=0
FileUpperToLowerCase=0
FolderAlignExtensions=0
FolderUpperCase=0
FullScreen=0
StatusColumnWidths=0,0,0,0
StatusColumns=NR,SCEF,D,T
```
![Screenshot_20220326_073207](https://user-images.githubusercontent.com/284112/160224769-6f001fe8-feb6-44da-9eae-34429d4c0c76.png)

**Comment by Guest007** [comment]
По той же причине переделал «Альтернативный полный» вот так:
```
[Panel/ViewModes/Mode0]
AlignExtensions=0
ColumnWidths=0,10,10,0
Columns=NM,SCEF,D,T
FileLowerCase=0
FileUpperToLowerCase=0
FolderAlignExtensions=0
FolderUpperCase=0
FullScreen=0
StatusColumnWidths=0,0,8,8,0
StatusColumns=N,S,O,U,A
```
O - Owner, U - Group. Про U в хелпе не написано.
Кстати, добиться отображения даты/времени в одной колонке не получилось - время съезжает вправо за границу панели. Собирал из гита 29.03.22
И, может кто подскажет, можно ли включить lynx-motion style как в mc? 
![изображение](https://user-images.githubusercontent.com/147067/161009875-502f2cc5-b9d5-4a4e-8330-cca753ad287f.png)

**Reply by bigvax** [reply]
Привет!
> И, может кто подскажет, можно ли включить lynx-motion style как в mc?

В файле ~/.config/far2l/settings/key_macros.ini
добавь строки:

```
[KeyMacros/Shell/Left]
Description=Lynx style motion: left
DisableOutput=0x1
EmptyCommandLine=0x1
Sequence=CtrlPgUp

[KeyMacros/Shell/Right]
Description=Lynx style motion: right
DisableOutput=0x1
EmptyCommandLine=0x1
Sequence=CtrlPgDn
```

Подробнее о полезных макросах я писал тут: [https://github.com/elfmz/far2l/discussions/1146](https://github.com/elfmz/far2l/discussions/1146)
